src/nvim/charset.c

```diff
@@ -1375,7 +1375,7 @@ void getvvcol(win_T *wp, pos_T *pos, colnr_T *start, colnr_T *cursor,
     ptr = ml_get_buf(wp->w_buffer, pos->lnum, false);
 
     if (pos->col < (colnr_T)STRLEN(ptr)) {
-      int c = (*mb_ptr2char)(ptr + pos->col);
+      int c = utf_ptr2char(ptr + pos->col);
```

Comment:
Add `const` unless this changes in a future vim-patch.

Reply:
I didn't see any coding style guideline in the wiki requiring `const` on variables which are not pointers.
I don't have a strong objection to adding `const` in cases such as this, but i don't see much benefit in it.

Reply:
Especially with a small local block there is really not much benefit to add const to a local variable binding (it just trivially repeats info already visible in the block), we should focus the effort to pointer targets (where const makes it a distinct type), and possibly bindings in really large scopes.